TShock.Modifications.Bootstrapper/Program.cs

```diff
@@ -15,15 +15,30 @@ static void Main(string[] args)
 			string sourceAsm = null;
 			string modificationGlob = null;
 			string outputPath = null;
+			string folder = null;
```

Comment:
Shouldn't this be defaulted to something?

Reply:
Only the one question, otherwise lgtm

Reply:
I have no idea

Reply:
I agree

Reply:
(w.r.t. "I have no idea")

Reply:
It seems to have been defaulted to "Debug" before the change. Should that be the default value?

Reply:
The csproj has a build task that generates env.config with the current configuration set:
https://github.com/Pryaxis/TerrariaAPI-Server/blob/1f1901ac9dc891cd4663585a2b61a6179726407f/TShock.Modifications.Bootstrapper/TShock.Modifications.Bootstrapper.csproj#L100
`folder` should refer to the current configuration when building via anything that uses the csproj